.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  checks:
+    name:  TOXENV=${{ matrix.tox-env }}
```

Comment:
nit: there are two spaces between key and value instead of one

Reply:
Fixed.